backend/.env.example

```
@@ -1,2 +1,6 @@
 OIDC_CLIENT_ID=""
 OIDC_CLIENT_SECRET=""
+PRIVATE_KEY="
+-----BEGIN RSA PRIVATE KEY-----
+<KEY>
+"
```

Would be a bit nicer to have the ending block included as well.
```suggestion
PRIVATE_KEY="
-----BEGIN RSA PRIVATE KEY-----
<KEY>
-----END RSA PRIVATE KEY-----
"
```